### DISCUSSION TITLE
disable configuring the Management0 interface and the management VRF on leafs with inband mgmt

### DISCUSSION THREAD

**Discussion by kmueller68** [aristanetworks/avd#3397] [Q&A]
Hello.

In the AVD campus fabric example is exmplained how to disable generation of oob interfaces on leafs.

In Campus Networks, having a dedicated out-of-band management network in each IDF is uncommon. Therefore, you can easily disable configuring the Management0 interface and the management VRF by adding these variables to the DC1_LEAFS.yml group_vars.
```
mgmt_gateway: null
mgmt_interface: null
mgmt_interface_vrf: default
```

Should this really work?
I have tested it but the `interface Management1`is still created including an IP?

```
---
### group_vars/DC1_LEAFS.yml

type: leaf     # Must be leaf
mgmt_gateway: null
mgmt_interface: null
mgmt_interface_vrf: default
```

thank you
Karlheinz

**Comment by carlbuchmann** [comment]
Thanks for raising this, @kmueller68 - I have created issue #3400 and added it to one of the initiatives to improve Campus support in AVD.
Please look at this other issue, #3158, for more information on how we plan to improve inband management features.
